build/build.sh

```diff
@@ -26,8 +26,8 @@ if [ -z "${ARCH}" ]; then
     exit 1
 fi
 if [ -z "${VERSION}" ]; then
-    echo "VERSION must be set"
-    exit 1
+    VERSION=$(git describe --tags --always --dirty)
```

Comment:
Humm, I thought VERSION would be passed in when calling this build.sh?
https://github.com/kubernetes/dns/blob/bdeeba54377008acd635f184b97dd035a0916f68/rules.mk#L105-L109

Reply:
Yes, it should be. For some reason it is not and i am not sure where to add a log to debug it.
I am thinking https://github.com/kubernetes/dns/blob/bdeeba54377008acd635f184b97dd035a0916f68/Makefile#L48 did not work, but not entirely sure why that would be. So i added that same command here to see if this works.

Reply:
Got it, yea let's give this a try.